check_release.py

```diff
@@ -160,4 +145,4 @@ def main():
 
```

Comment:
> don't forget to tag the release ..

Not that this ever really reminded someone, but it can definitely go now. :)

Reply:
Removed in 47869c0